src/lsstseries/analysis/structure_function/base_calculator.py

```diff
@@ -146,8 +146,8 @@ def _calculate_binned_statistics(self, sample_values=None, statistic_to_apply="m
                 )
 
             # return the mean delta_time values for each bin
-            # bin_means, _, _ = binned_statistic(self._dts, self._dts, statistic="mean", bins=self._bins)
-            return [(bin_edgs[0:-1] + bin_edgs[1:]) / 2], [sfs]
+            bin_means, _, _ = binned_statistic(self._dts, self._dts, statistic="mean", bins=self._bins)
+            return [bin_means], [sfs]
```

Comment:
Should the docstring for this function be updated to specify that it returns the `bin_means` now? I see it still mentions the edges (but that might be for another reason)

Reply:
Good call out, I'll update the docstring.

Reply:
Updated